Kernel/Syscalls/execve.cpp

```diff
@@ -215,30 +215,32 @@ static ErrorOr<FlatPtr> get_load_offset(const ElfW(Ehdr) & main_program_header,
     if (main_program_header.e_type != ET_EXEC)
         return EINVAL;
 
```

Comment:
> Tim Schumacher wrote most of the patch, [...]

`Co-Authored-By: Tim Schumacher <timschumi@gmx.de>`? :^)

Reply:
Forgout about this, I'll add this.